### DISCUSSION TITLE
why is LosslesCut faster that using ffmpeg?

### DISCUSSION THREAD

**Discussion by user4302** [mifi/lossless-cut#917] [Q&A]
so i tried converting an FLV to MP4 and using ffmpeg on the CLI and it was taking ages, while using LosslessCut took mere seconds.
firstly this is amazing xD
secondly why is this? am i doing somehitng wrong? i just used 'ffmpeg input.flv output.mp4'

**Comment by mifi** [comment]
hi! because `ffmpeg input.flv output.mp4` will re-encode it to h264, which is slow. if you want to do the same as losslesscut does but from command line, do this: `ffmpeg -i input.flv -c copy output.mp4 `

**Reply by mifi** [reply]
or check in the help page after running a cut, it will log all the ffmpeg commands there
